ci-operator/config/opendatahub-io/kueue/opendatahub-io-kueue-dev.yaml

```diff
@@ -34,7 +37,8 @@ tests:
           oc apply --server-side -k "github.com/opendatahub-io/kueue/config/rhoai?ref=dev"
```

Comment:
I'm a bit confused here on the manifests that would be present during the PJ rehearsal. Would it make sense to just get rid of this conditional all together? @sutaakar

Reply:
This condition is here to specify what kueue code is used for PJ rehearsal runs.
As the PJ rehearsal doesn't run against PR then there is used manifest from default branch.